### PR TITLE
Use FuzzySearch to search access links

### PR DESCRIPTION
**Pull request by VictoriousRaptor**
## Feature

Use `StringMatcher.FuzzySearch()` to search Access links in Explorer plugin to improve search results, like allow searching in pinyin.

## Tests

- Access links can be fuzzy-searched.

### CODE REVIEW

File: Plugins/Flow.Launcher.Plugin.Explorer/Search/QuickAccessLinks/QuickAccess.cs [hunk 1]
```diff
@@ -13,20 +14,17 @@ internal static List<Result> AccessLinkListMatched(Query query, IEnumerable<Acce
             if (string.IsNullOrEmpty(query.Search))
                 return new List<Result>();
 
-            string search = query.Search.ToLower();
-
-            var queriedAccessLinks =
-                accessLinks
-                .Where(x => x.Name.Contains(search, StringComparison.OrdinalIgnoreCase) || x.Path.Contains(search, StringComparison.OrdinalIgnoreCase))
+            return accessLinks
+                .Where(x => StringMatcher.FuzzySearch(query.Search, x.Name).IsSearchPrecisionScoreMet() || StringMatcher.FuzzySearch(query.Search, x.Path).IsSearchPrecisionScoreMet())
```
Comment (taooceros) [inline Plugins/Flow.Launcher.Plugin.Explorer/Search/QuickAccessLinks/QuickAccess.cs:18]: maybe use the api version? Is there a specific reason that we didn't do this before? @jjw24

Comment (VictoriousRaptor) [reply Plugins/Flow.Launcher.Plugin.Explorer/Search/QuickAccessLinks/QuickAccess.cs]: Using API version now. imo fuzzy search and pinyin search is better anyways.

Comment (taooceros) [reply Plugins/Flow.Launcher.Plugin.Explorer/Search/QuickAccessLinks/QuickAccess.cs]: Yeah we should

Comment (jjw24) [reply Plugins/Flow.Launcher.Plugin.Explorer/Search/QuickAccessLinks/QuickAccess.cs]: >Is there a specific reason that we didn't do this before? @jjw24

Not that I can recall right now

Comment (jjw24) [reply Plugins/Flow.Launcher.Plugin.Explorer/Search/QuickAccessLinks/QuickAccess.cs]: @VictoriousRaptor did you want to bump version first before merge?

**Review by jjw24 — APPROVED** [review]

File: Plugins/Flow.Launcher.Plugin.Explorer/Search/QuickAccessLinks/QuickAccess.cs (hunk 1 above)
Comment (VictoriousRaptor) [reply Plugins/Flow.Launcher.Plugin.Explorer/Search/QuickAccessLinks/QuickAccess.cs]: Ok I'll bump it.

**Review by jjw24 — APPROVED** [review]

**Pull request merged**